### ISSUE TITLE
A big button to zero out all the campaign immediatelly !

### ISSUE THREAD

**Issue by theochino** [ProgressiveCoders/Spoke#890]
When inside a campaign with more than several hundred texters and after accidentally splitting across all the texters, a way to zero out everyone in one shot.

**Comment by ibrand** [comment]
@deasterdaywfp I just saw you commenting on the Add All button and so I thought of you for this one. Would this be more useful than add all?

@jeffm2001 does this come up for you?

**Comment by jeffm2001** [comment]
Not really, since we primarily use dynamic assignment. I guess I'm not 100% clear on the problem we're trying to solve here. If the mistake was made before saving I would just refresh the page to discard the changes, and otherwise prolly use the message review to do the reassignment in bulk.
